apps/modernization-ui/src/apps/search/laboratory-report/labReportFormTypes.ts

```diff
@@ -34,7 +37,8 @@ type General = {
     pregnancyStatus?: Selectable;
     processingStatus?: Selectable[];
     programAreas?: Selectable[];
-};
+    providerSearch?: LabReportProviderSearch | null;
```

Comment:
issue:  use the properties that were added for the specific facility/ provider.  You can display the appropriate one when the type is selected.